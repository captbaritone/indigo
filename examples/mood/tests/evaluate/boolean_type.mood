pub fn test(): bool {
  true
}