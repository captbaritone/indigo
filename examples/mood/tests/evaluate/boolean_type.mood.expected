-----------------
INPUT
----------------- 
pub fn test(): bool {
  true
}
-----------------
OUTPUT
-----------------
1